scripts/nlp/evaluate.py

```diff
@@ -1,5 +1,86 @@
 # all the evaluation related function like: accuracy calc, false positive etc
+import spacy
+from spacy.gold import GoldParse
+from spacy.scorer import Scorer
+
+def evaluate(nlp, examples, ent='LOC_ACCIDENT'):
+    scorer = Scorer()
```

Comment:
calculate score manually, means create your own functions to calculate the recall, precision etc